Documentation/CRDs/Cluster/external-cluster.md

```diff
@@ -84,6 +87,20 @@ See the [Multisite doc](https://docs.ceph.com/en/quincy/radosgw/multisite/#confi
 python3 create-external-cluster-resources.py --rbd-data-pool-name <pool_name> --format bash --rgw-endpoint <rgw_endpoint> --rgw-realm-name <rgw_realm_name>> --rgw-zonegroup-name <rgw_zonegroup_name> --rgw-zone-name <rgw_zone_name>>
 ```
 
+### Topology Based Provisioning
+
+Enable Topology Based Provisioning for RBD pools by passing `--topology-pools`, `--topology-failure-domain-label` and `--topology-failure-domain-values` flags.
+A new storageclass will be created by the import script named `ceph-rbd-topology` with `volumeBindingMode: WaitForFirstConsumer`
+and will configure topologyConstrainedPools according the input provided.
+Later use the storageclass to create a volume in the pool matching the topology of the pod scheduling.
+
+```console
+python3 create-external-cluster-resources.py --rbd-data-pool-name pool_name --topology-pools p,q,r --topology-failure-domain-label labelName --topology-failure-domain-values x,y,z --format bash
+```
+
+!!! note
+    In the list of --topology-pools we can pass any replica pools
```

Comment:
Since there are no restrictions on the pools, I'd suggest no need for this note.